src/modules/auth/dtos/loginUser.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Exclude } from 'class-transformer';
+import { IsEmail, IsString } from 'class-validator';
+
+export class LoginUserDto {
+  @IsString()
+  @IsEmail()
+  readonly email: string;
```

Comment:
look like eslint not work. there should be empty line

Reply:
Updated